### ISSUE TITLE
[Chore] Improve the way to handle uploaded CSV file

### ISSUE THREAD

**Issue by longnd** [aungkoko1234/web-scrapper-backend#20]
Issue
---
Currently, after a CSV file is uploaded, the application itself uploads the file again to S3, then streams the file to parse.
Isn't it too cumbersome, why doesn't the application parse the file locally, on the web server, after it was uploaded by the user?

**Comment by aungkoko1234** [comment]
Yeah it is inefficient. At first I was planning to  upload the files to s3 and just pass the file path to queue. But I implemented wrongly. Due to the current implementation, it should be better with parsing the file locally.

**Comment by longnd** [comment]
> At first I was planning to upload the files to s3 and just pass the file path to queue. But I implemented wrongly

even with this solution, it is still inefficient, IMO. Why does the application need to upload the file to S3 to process?
It leads to extra job that the application need to remove the file from S3 after complete the parsing. 
why doesn't it just parse the file locally?

**Comment by aungkoko1234** [comment]
I have removed the uploading s3 file in pull request #28  @longnd
